knowledge/animals/cloning/dolly_the_sheep/qna.yaml

```diff
@@ -0,0 +1,29 @@
+created_by: 
+domain: genetics
+seed_examples:
+- answer: |
+Dolly the sheep was a female Finn-Dorset sheep and the first mammal that was cloned from an adult somatic cell
+  question: Who was dolly the sheep?
+- answer: |
+Her cloning proved that a cloned organism could be produced from a mature cell from a specific body part.
+  question: What has proven clonning Dolly the sheep? 
+- answer: |
+Contrary to popular belief, she was not the first animal to be cloned.
+  question: What is a common misconception about cloning animals?
+- answer: |
+Dolly was cloned by Keith Campbell, Ian Wilmut and colleagues at the Roslin Institute, 
+part of the University of Edinburgh, Scotland, and the biotechnology company PPL Therapeutics,
+ based near Edinburgh. 
+  question: Who cloned Dolly the sheep?
+- answer: |
+On 14 February 2003
+  question: When Dolly the sheep died?
+- answer: |
+Dolly was euthanised because she had a progressive lung disease and severe arthritis.
+  question: Why was Dolly the sheep euthanised?
+task_description: ' '
```

Comment:
Please provide a task description as this is used by the synthetic data generation process.